includes/features/SU-SWS/stanford_news/stanford_news.feature

```diff
@@ -74,10 +71,10 @@
     Then I uncheck the box "path[pathauto]"
     Then I fill in "edit-path-alias" with "[random]"
     Then I press the "Save" button
```

Comment:
If there is a  'press the "save" button' in a scenario then it is destructive. Please change this to @dev and @destructive